src/errors/Forbidden.ts

```diff
@@ -1,9 +1,10 @@
 import httpStatus from 'http-status';
+import type { i18n as Ii18n } from 'i18next';
 import APIError from './APIError';
 
 class Forbidden extends APIError {
-  constructor() {
-    super('You are not allowed to perform this action.', httpStatus.FORBIDDEN);
+  constructor(i18n?: Ii18n) {
+    super(i18n.t('error:notAllowedToPerformAction'), httpStatus.FORBIDDEN);
```

Comment:
Should there be a fallback here, similar to the other `APIError` classes?
```suggestion
    super(i18n?.t('error:notAllowedToPerformAction') || 'You are not allowed to perform this action.', httpStatus.FORBIDDEN);
```